src/main/java/org/checkerframework/specimin/MustImplementMethodsVisitor.java

```diff
@@ -184,6 +265,7 @@ private boolean overridesAnInterfaceMethodImpl(
       for (String name : typeParametersMap.getNames()) {
         String interfaceViewpointName = name.substring(name.lastIndexOf('.') + 1);
         String localViewpointName = typeParametersMap.getValueBySignature(name).get().describe();
+        localViewpointName = Pattern.quote(localViewpointName);
```

Comment:
A comment justifying why quoting this is necessary will be helpful later, so that we don't accidentally delete it.